src/main/java/github/weichware10/analyse/gui/HeatmapConfigurator.java

```diff
@@ -0,0 +1,52 @@
+package github.weichware10.analyse.gui;
+
+import github.weichware10.analyse.config.HeatmapConfig;
+import github.weichware10.analyse.gui.util.AbsScene;
+import javafx.event.ActionEvent;
+import javafx.scene.control.Button;
+import javafx.scene.control.ButtonBar.ButtonData;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Dialog;
+import javafx.scene.layout.VBox;
+
+/**
+ * Konfiguratorfenster für Heatmap-Analysen der App.
+ */
+public class HeatmapConfigurator extends AbsScene {
+
+    private static VBox root;
+    private static HeatmapConfiguratorController controller;
+
+    /**
+     * Startet die App.
+     *
+     * @return - Erstellte Konfiguration
+     */
+    public static void start(HeatmapConfig hmConfig) {
+        Dialog<String> configDialog = new Dialog<>();
+        ButtonType applyButtonType = new ButtonType("Übernehmen", ButtonData.APPLY);
+        configDialog.getDialogPane().getButtonTypes().addAll(applyButtonType, ButtonType.CANCEL);
+
+        configDialog.setTitle("Konfiguration Heatmap");
+
+        InitResult ir = initialize(HeatmapConfigurator.class.getResource("HeatmapConfigurator.fxml"));
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 102).